tests/rpc.rs

```diff
@@ -1,40 +1,18 @@
-use beerus::{
-    config::MAINNET_STARKNET_CHAINID,
-    gen::{
-        Address, BlockHash, BlockId, BlockNumber, BlockTag,
-        BroadcastedInvokeTxn, BroadcastedTxn, Felt, FunctionCall,
-        GetBlockWithTxHashesResult, GetBlockWithTxsResult, GetClassAtResult,
-        GetClassResult, GetTransactionByBlockIdAndIndexIndex, InvokeTxn,
-        InvokeTxnV1, InvokeTxnV1Version, PriceUnit, Rpc, StorageKey,
-        SyncingResult, Txn, TxnExecutionStatus, TxnHash, TxnReceipt,
-        TxnReceiptWithBlockInfo, TxnStatus,
-    },
+use beerus::gen::{
+    Address, BlockHash, BlockId, BlockNumber, BlockTag, BroadcastedInvokeTxn,
+    BroadcastedTxn, Felt, FunctionCall, GetBlockWithTxHashesResult,
+    GetBlockWithTxsResult, GetClassAtResult, GetClassResult,
+    GetTransactionByBlockIdAndIndexIndex, InvokeTxn, InvokeTxnV1,
+    InvokeTxnV1Version, PriceUnit, Rpc, StorageKey, SyncingResult, Txn,
+    TxnExecutionStatus, TxnHash, TxnReceipt, TxnReceiptWithBlockInfo,
+    TxnStatus,
 };
 
 mod common;
+mod starknet;
 
 use common::err::Error;
 
-#[tokio::test]
```

Comment:
Why remove these test cases?

Reply:
I guess formatting issue O.o. Returning it